This patch fixes building xmloff on openSUSE 10.3 x86_64, error

    /usr/lib64/gcc/x86_64-suse-linux/4.2.1/../../../../x86_64-suse-linux/bin/ld: ../unxlngx6.pro/slo/layerexport.o: relocation R_X86_64_PC32 against `exportXForms(SvXMLExport&)' can not be used when making a shared object; recompile with -fPIC

gcc used

    15:40:36 janneke@peder:~/vc/ooo-m241
    $ gcc --version
    gcc (GCC) 4.2.1 (SUSE Linux)

--- xmloff/inc/xformsexport.hxx~	2008-01-03 14:45:07.000000000 +0100
+++ xmloff/inc/xformsexport.hxx	2008-01-03 15:23:59.000000000 +0100
@@ -49,12 +49,12 @@
 
 
 /** export an XForms model. */
-void SAL_DLLPRIVATE exportXForms( SvXMLExport& );
+void XMLOFF_DLLPUBLIC exportXForms( SvXMLExport& );
 
-rtl::OUString SAL_DLLPRIVATE getXFormsBindName( const com::sun::star::uno::Reference<com::sun::star::beans::XPropertySet>& xBinding );
+rtl::OUString XMLOFF_DLLPUBLIC getXFormsBindName( const com::sun::star::uno::Reference<com::sun::star::beans::XPropertySet>& xBinding );
 
-rtl::OUString SAL_DLLPRIVATE getXFormsListBindName( const com::sun::star::uno::Reference<com::sun::star::beans::XPropertySet>& xBinding );
+rtl::OUString XMLOFF_DLLPUBLIC getXFormsListBindName( const com::sun::star::uno::Reference<com::sun::star::beans::XPropertySet>& xBinding );
 
-rtl::OUString SAL_DLLPRIVATE getXFormsSubmissionName( const com::sun::star::uno::Reference<com::sun::star::beans::XPropertySet>& xBinding );
+rtl::OUString XMLOFF_DLLPUBLIC getXFormsSubmissionName( const com::sun::star::uno::Reference<com::sun::star::beans::XPropertySet>& xBinding );
 
 #endif
